src/styles/pages/services/services.module.scss

```diff
@@ -0,0 +1,178 @@
+.mainContainer {
+    font-family: Arial; //import later 
+    display: flex;
+    flex-direction: column;
+}
+
+.servicesContainer {
+    background-color: var(--color-white);
+    color: var(--color-maroon);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
These properties aren't needed since they aren't contributing too many styles. You can just keep `padding` and `row-gap`